addons/website_slides/models/slide_slide.py

```diff
@@ -183,6 +183,7 @@ class Slide(models.Model):
     nbr_webpage = fields.Integer("Number of Webpages", compute='_compute_slides_statistics', store=True)
     nbr_quiz = fields.Integer("Number of Quizs", compute="_compute_slides_statistics", store=True)
     total_slides = fields.Integer(compute='_compute_slides_statistics', store=True)
+    website_published = fields.Boolean(tracking=True)
```

Comment:
Website published is a computed field, it should probably not be tracked. I would track is_published instead which is the real field storing the boolean value. Website published is a related or a computed based on published + current website (in case of multi website publish mixin).